wordleBattleProject/core/src/com/mygdx/game/view/SingleplayerGameView.java

```diff
@@ -1,26 +1,93 @@
 package com.mygdx.game.view;
 
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.GL20;
 import com.badlogic.gdx.graphics.g2d.BitmapFont;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
 import com.mygdx.game.exception.StateException;
+import com.mygdx.game.model.keyboard.KeyboardInput;
 import com.mygdx.game.model.states.PlayState;
 import com.mygdx.game.model.states.State;
 
-public class SingleplayerGameView extends View{
+public class SingleplayerGameView extends View {
 
-    BitmapFont font;
+    private final float buttonWidth = Gdx.graphics.getWidth() / 16f;
+    private final float buttonHeight = Gdx.graphics.getHeight() / 20f;
+    private final float buttonPadding = buttonWidth / 5f;
 
-    public SingleplayerGameView(){
-        font = new BitmapFont();
-        font.getData().setScale(6,6);
+    private final String[][] buttonValues = {
```

Comment:
Should be moved to PlayState, and be fetched from there.

Reply:
Skal jeg slette den packagen som heter keyboard da? Blir jo ganske tomt i KeyboardInput